components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -208,6 +213,13 @@ func (c *remoteCmdExecutor) executeCmdOnNode(command string, inputFiles map[stri
 	if errorCheckEnableInOutput {
 		err = checkResultOutputForError(output)
 	}
+
```

Comment:
this error check is duplicate to below error check

Reply:
Done